test/libsolidity/ASTJSON/assembly/inline_assembly_memory_safe_parseOnly.json

```diff
@@ -0,0 +1,144 @@
+{
+  "absolutePath": "a",
+  "id": 9,
+  "nodeType": "SourceUnit",
+  "nodes":
+  [
+    {
+      "abstract": false,
+      "baseContracts": [],
+      "contractDependencies": [],
+      "contractKind": "contract",
+      "id": 8,
+      "name": "C",
+      "nameLocation": "9:1:1",
+      "nodeType": "ContractDefinition",
+      "nodes":
+      [
+        {
+          "body":
+          {
+            "id": 6,
+            "nodeType": "Block",
+            "src": "57:136:1",
+            "statements":
+            [
+              {
+                "AST":
+                {
+                  "nodeType": "YulBlock",
+                  "src": "111:78:1",
+                  "statements":
+                  [
+                    {
+                      "body":
+                      {
+                        "nodeType": "YulBlock",
+                        "src": "133:50:1",
+                        "statements":
+                        [
+                          {
+                            "body":
+                            {
+                              "nodeType": "YulBlock",
+                              "src": "157:3:1",
+                              "statements": []
+                            },
+                            "name": "f2",
+                            "nodeType": "YulFunctionDefinition",
+                            "src": "143:17:1"
+                          },
+                          {
+                            "nodeType": "YulAssignment",
+                            "src": "169:6:1",
+                            "value":
+                            {
+                              "kind": "number",
+                              "nodeType": "YulLiteral",
+                              "src": "174:1:1",
+                              "type": "",
+                              "value": "2"
+                            },
+                            "variableNames":
+                            [
+                              {
+                                "name": "x",
+                                "nodeType": "YulIdentifier",
+                                "src": "169:1:1"
+                              }
+                            ]
+                          }
+                        ]
+                      },
+                      "name": "f1",
+                      "nodeType": "YulFunctionDefinition",
+                      "src": "119:64:1"
+                    }
+                  ]
+                },
+                "documentation": "@solidity memory-safe-assembly",
+                "evmVersion": %EVMVERSION%,
+                "externalReferences": [],
+                "id": 5,
+                "memorySafe": false,
```

Comment:
This is a problem...

Reply:
On the other hand: since the documentation node is in here, maybe I don't need to add this to the JSON AST specifically at all.